### DISCUSSION TITLE
Local send minimized tray icon not showing!!

### DISCUSSION THREAD

**Discussion by SDee001** [localsend/localsend#1192] [Discussion (English, International)]
Local send minimized tray icon not showing when downloaded from flathub from linux mint software manager

**Comment by null-von-sushi** [comment]
I have the same issue, also using the flatpak version but on Endeavour OS (Arch based) with KDE desktop

**Comment by steakhutzeee** [comment]
Same here, Opensuse Tumbleweed and KDE.

**Comment by thegabriele97** [comment]
Same here, flatpak version on ubuntu 22.04 gnome, pop os 22.04 pop shell and on my steam deck (steam os latest version, kde)

**Comment by SDee001** [comment]
its fixed for me now as i downloaded it as a .deb in my pc
if this solved it for you guys we will close this as solved

**Comment by BB-PawStudioNet** [comment]
I have the same issue under Kubuntu 24.04 if I run it under my normal user just an empty space beside the bluetooth icon, but If I start it as a root, an icon show up as localsend.

This is the flatpack version, the downloaded deb version doesn't want run, but t the moment I have no idea why...

normal:
![image_2024-09-23_115913942](https://github.com/user-attachments/assets/1d851e30-0944-4909-ad05-29a6125e530a)

root:
![image_2024-09-23_120000331](https://github.com/user-attachments/assets/17c8f18c-8fe1-4e17-bcb7-0543067d97b3)
